docs/source/building.rst

```diff
@@ -167,10 +167,26 @@ Regression Manager
 
 .. envvar:: COVERAGE
 
-    Enable to report Python coverage data. For some simulators, this will also report :term:`HDL` coverage.
+    Enable to collect Python coverage data for user code.
+    For some simulators, this will also report :term:`HDL` coverage.
```

Comment:
Is this true?

Reply:
It's true for Riviera: https://github.com/cocotb/cocotb/blob/17038c0ff372423f64282146a90c8d73cf021b8d/src/cocotb_tools/makefiles/simulators/Makefile.riviera#L87

Came in 11 years ago in https://github.com/cocotb/cocotb/commit/71550332dfa4e5bf6a3dd3ec64a2241cdb0dadd1

I had actually planned to deprecate this together with other envvars in #1495; 1.9/2.0 seems like a good point in time to finally do it.